module/zfs/btree.c

```diff
@@ -2171,3 +2171,9 @@ zfs_btree_verify(zfs_btree_t *tree)
 		return;
 	zfs_btree_verify_poison(tree);
 }
+
+/* BEGIN CSTYLED */
+ZFS_MODULE_PARAM(zfs, zfs_, btree_verify_intensity, UINT, ZMOD_RW,
```

Comment:
Can you double check that the intensity can be safely modified at run time.  Based on the comments this appears to be the case, but it's certainly not something I've ever verified.

It's also not clear to me adding this module option will be particularly useful.  If we suspect an issue with the btree implementation it's probably not unreasonable for a developer to modify the code and rebuild from source.  That said, I'm not deadset against adding this.  Goodness know there are already enough other module options.

Reply:
The checks should just activate like they do when a user sets zfs_flags. These two issues made me suspect that it might be useful to be able to ask end users to turn this on:

https://github.com/openzfs/zfs/issues/11893
https://github.com/openzfs/zfs/issues/13445